yarn.lock

```diff
@@ -6955,12 +6953,10 @@ test-exclude@^6.0.0:
     minimatch "^3.0.4"
 
 "test-package-a@link:./packages/playground/nested-deps/test-package-a":
-  version "0.0.0"
-  uid ""
+  version "2.0.0"
 
 "test-package-b@link:./packages/playground/nested-deps/test-package-b":
-  version "0.0.0"
-  uid ""
+  version "1.0.0"
```

Comment:
Uhm... why is renovate touching this?

Reply:
I don't know why it is doing it in this particular PR, but we should configure it to ignore them. These are hand written for testing